pluma/dialogs/pluma-preferences-dialog.h

```diff
@@ -81,6 +81,9 @@ GType		 pluma_preferences_dialog_get_type	(void) G_GNUC_CONST;
 
 void		 pluma_show_preferences_dialog		(PlumaWindow *parent);
 
+static       gboolean pluma_preferences_dialog_page_scroll_event_cb (GtkWidget      *notebook,
+                                                                     GdkEventScroll *event);
+
```

Comment:
Remove `pluma_preferences_dialog_page_scroll_event_cb` since this function is used in only one .c file.